src/components/application_manager/test/commands/mobile/put_file_test.cc

```diff
@@ -344,6 +345,53 @@ TEST_F(PutFileRequestTest, Run_InvalidPutFile_UNSUCCESS) {
   command->Run();
 }
 
+TEST_F(PutFileRequestTest, Run_SrcSumEqual_SendSuccessResponse) {
+  (*msg_)[am::strings::msg_params][am::strings::crc32_check_sum] = 2768625435u;
+
+  ExpectReceiveMessageFromSDK();
+  ON_CALL(app_mngr_, get_settings())
+      .WillByDefault(ReturnRef(app_mngr_settings_));
+  ON_CALL(app_mngr_settings_, app_storage_folder())
+      .WillByDefault(ReturnRef(kStorageFolder));
+  ON_CALL(*mock_app_, folder_name()).WillByDefault(Return(kFolder));
+  const uint32_t avaliable_space = binary_data_.size() + 1;
```

Comment:
@BSolonenko typo + type `size_t`

Reply:
@AByzhynar Fixed in https://github.com/smartdevicelink/sdl_core/pull/2139/commits/17cae31dfeb5a8e55281633ebe3efc6dd051c614